Create DAOsign

TetianaRiabova opened the pull request:
# Grant Application Checklist

- [ ] The [`applications/application-template.md`](../applications/application-template.md) or [`applications/research-template.md`](../applications/research-template.md) has been copied, renamed ( "project_name.md") and updated.
- [ ] An Ethereum address for the payment of the milestones is provided inside the application.  
- [ ] The total funding amount of the project is up to USD $30k for every grant applications.
- [ ] The initial PR contains only one commit (squash if needed before submitting your PR).

mongelly commented:
Thank you for submitting the application. 
Before starting to review your project, we hope that you will supplement the information according to the template, such as Technology stack，Milestone Details，Cost and more.

mongelly commented:
Hi @TetianaRiabova, can you update the roadmap, we need to know somethings about,

TetianaRiabova commented:
Hello @mongelly, 
I hope that you're doing well! A while ago, applied to the Vechain Grant Program, and since that time, our product has advanced a lot.

Since we have already received a few grants from Mintbase, and chains like Polkadot or SUI, we believe, it's a perfect time to discuss the potential collaboration and deployment on your chain. I have updated the application form and added more information. Please let me know if you have any questions.

Also, I would like to offer a call with our founder, who can provide a demo of our platform to you. What do you think about it?

libotony commented:
> Hello @mongelly, I hope that you're doing well! A while ago, we applied to the Vechain Grant Program, and since that time, our product has advanced a lot.
> 
> Since we have already received a few grants from Mintbase, and chains like Polkadot or SUI, we believe, it's a perfect time to discuss the potential collaboration and deployment on your chain. I have updated the application form and added more information. Please let me know if you have any questions.
> 
> Also, I would like to offer a call with our founder, who can provide a demo of our platform to you. What do you think about it?

@TetianaRiabova, thank you for the update. We've recently launched a new [website](https://www.vechain.org/grant-program/) and introduced a new process for our grant program. Could you please submit your application through the new system? Our team will then get in touch with you.

libotony commented:
> @libotony Thank you for letting me know. We will apply for the new grant program, then! But we still would be happy to demo our product to you. Thanks once again! 😊

Not at all, wish you good luck! I'll close this PR.
